charts/primary-site/templates/cronjobs/garbage-collector.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   failedJobsHistoryLimit: {{ .Values.garbageCollector.failedJobsHistoryLimit }}
   successfulJobsHistoryLimit: {{ .Values.garbageCollector.successfulJobsHistoryLimit }}
   jobTemplate:
+    metadata:
+      labels:
+        app: garbage-collector
```

Comment:
@james-rms heads up I had to add this label to make it possible to find completed garbage collector jobs